### ISSUE TITLE
Information regarding the training

### ISSUE THREAD

**Issue by ajay1234567899** [hasanirtiza/Pedestron#154]
Hello sir,

In table 4 of the paper, on the city person dataset, a number of 11.2 was obtained.

To obtain that result, you have used an hrnet backbone which was pretrained on the imagenet, and then trained the cascade mask rcnn from scratch right?(Jointly training the backbone+detection network).

Or any pre-trained weight of coco was used to fine-tune the results?

Do let me know which of the above point is correct.

Thanks for your time

**Comment by hasanirtiza** [comment]
> To obtain that result, you have used an hrnet backbone which was pretrained on the imagenet, and then trained the cascade mask rcnn from scratch right?(Jointly training the backbone+detection network).

I would say yes.
